x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/ShardFollowTasksExecutor.java

```diff
@@ -158,7 +181,8 @@ protected void innerUpdateMapping(long minRequiredMappingVersion, LongConsumer h
                             return;
                         }
                         MappingMetadata mappingMetadata = indexMetadata.mapping();
-                        PutMappingRequest putMappingRequest = CcrRequests.putMappingRequest(followerIndex.getName(), mappingMetadata);
+                        PutMappingRequest putMappingRequest = CcrRequests.putMappingRequest(followerIndex.getName(), mappingMetadata)
+                            .masterNodeTimeout(UNBOUNDED_TIMEOUT);
```

Comment:
Should we add this to `CcrRequests.putMappingRequest` instead? There is one other caller of that method that does something similar (setting it to 30 minutes, mainly because that felt like "unbounded" at the time)

Reply:
++. Adjusted in https://github.com/elastic/elasticsearch/pull/60070/commits/6ad4d4fd4584485b7312bc0ec4f7e03e7af64082.